datadriven/src/test_data_reader.rs

```diff
@@ -10,10 +10,12 @@ pub struct TestDataReader<'a> {
     pub data: TestData,
     scanner: Enumerate<Lines<'a>>,
     pub logger: slog::Logger,
+    pub rewrite: bool,
+    pub rewrite_buffer: String,
```

Comment:
`Option<String>` seems a more safe definition. People can forget to check the Boolean flag.